### PR TITLE
Added a note about the implementation of the verbosity semantic methods

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | 2.6+ |
| Fixed tickets | #5123 |

### CODE REVIEW

File: components/console/introduction.rst [hunk 1]
```diff
@@ -223,6 +223,13 @@ verbosity levels::
         // ...
     }
 
+.. note::
+
+    For backwards compatibility reasons, these semantic methods do not exist in
```
Comment (cordoval) [inline components/console/introduction.rst]: do not exist/until -> are only introduced after in 3.0 to keep BC

File: components/console/introduction.rst [hunk 2]
```diff
@@ -223,6 +223,13 @@ verbosity levels::
...
+    the ``OutputInterface`` class until Symfony 3.0. They are defined in the
```
Comment (xabbuh) [inline components/console/introduction.rst]: "class" needs to be removed.

File: components/console/introduction.rst [hunk 3]
```diff
@@ -223,6 +223,13 @@ verbosity levels::
...
+    different implementations of the interface
+    (e.g. :class:`Symfony\\Component\\Console\\Output\\Output`).
```
Comment (xabbuh) [inline components/console/introduction.rst]: `Output` is a class

Comment (javiereguiluz) [inline components/console/introduction.rst]: I'm not sure I understand your comment. I try to say that these methods exist in the different implementations of the interface, for example in the `Output` class.

**Comment by weaverryan** [comment]
I like the note, and I like even more that these are in the interface for 3.0 :). Thanks Javier!

**Pull request closed without merging**